TFLite/src/tflite/__init__.py

```diff
@@ -0,0 +1,4 @@
+from os.path import dirname, basename, isfile, join
```

Comment:
Some readability feedback: Never import the individual functions. Always import a package or a module. In this case, you can just import path.

Also, it's advisable to import it like from <package> import <module>, for better readability and consistency.